packages/workflow/src/internals.ts

```diff
@@ -125,12 +125,15 @@ export class Activator implements ActivationHandler {
   /**
    * Mapping of update name to handler and validator
    */
-  readonly updateHandlers = new Map<string, { handler: WorkflowUpdateType; validator?: WorkflowUpdateValidatorType }>();
+  readonly updateHandlers = new Map<
+    string,
+    { handler: WorkflowUpdateType; validator?: WorkflowUpdateValidatorType; description?: string }
```

Comment:
We prefer avoiding adhoc types. Seems like that one slipped through the Workflow Update PR.

Please make this a named type instead; same for `signalHandlers` below.